CHANGELOG.md

```diff
@@ -17,7 +17,7 @@ Please avoid adding duplicate information across this changelog and JIRA/doc inp
 
 - ROX-14336: product `BuildDate` attribute was removed. It won't be returned by
 `/debug/versions.json` endpoint and `roxctl version --json` command.
-- ROX-12750: As announced in 3.73.0 (ROX-11101), some permissions for permission sets are being grouped for simplification. The deprecation process will remove and replace the deprecated permissions with the replacing permission as listed below. The access level granted to the replacing permission will be the lowest among all access levels of the replaced permissions. 
+- ROX-12750: As announced in 3.73.0 (ROX-11101), some permissions for permission sets are being grouped for simplification. The deprecation process will remove and replace the deprecated permissions with the replacing permission as listed below. The access level granted to the replacing permission will be the lowest among all access levels of the replaced permissions.
```

Comment:
I found following sentences hard to understand. 
- `The deprecation process will remove and replace the deprecated permissions with the replacing permission as listed below.`
- `The access level granted to the replacing permission will be the lowest among all access levels of the replaced permissions.`

Why:
- `The deprecation process` this is new entity that is not described anywhere or at least I don't know where :) 
- sentences are quite long 
- many repetitions of `permission`

How about something like this:
> This means that certain permissions will be removed and replaced with new ones as listed below. The new permission will have the lowest access level among all the permissions being replaced.